Where did he go? (#80)
wesbos/dad-jokes: readme.md

**A:** It had NoSQL.

---

> **Q:** The web developer was hungry, where did he go?

Reviewer: ```suggestion
**Q:** Where did the API go to eat?
```

> **A:** He went to the RESTaurant

Reviewer: ```suggestion
**A:** To the RESTaurant
```
